internal/api/docs/openapi.yaml

```diff
@@ -1669,3 +1669,24 @@ components:
         version:
           type: string
       type: object
+
+
+/v1/system/name:
```

Comment:
It might be better to use a more generic endpoint like /v1/system/information.
This would make the API more cohesive and easier to extend in the future if new properties are added that still fall under the general “information” category.

If you only need to retrieve the system name, you could either include a query parameter filter (e.g. ?filter=name) or use a more specific endpoint such as `/v1/system/information/name`.